Header header 
string[]  id     #joint ids 
float64[] data      #joint angle value
